CMakeLists.txt

```diff
@@ -99,8 +103,13 @@ project (Jerry CXX C ASM)
    get_filename_component(PATH_TO_GCC ${CMAKE_C_COMPILER} REALPATH)
    get_filename_component(DIRECTORY_GCC ${PATH_TO_GCC} DIRECTORY)
    get_filename_component(FILE_NAME_GCC ${PATH_TO_GCC} NAME)
-   string(REPLACE "gcc" "gcc-ar" CMAKE_AR ${FILE_NAME_GCC})
-   string(REPLACE "gcc" "gcc-ranlib" CMAKE_RANLIB ${FILE_NAME_GCC})
+   if("${PLATFORM}" STREQUAL "OPENWRT")
+    string(REPLACE "gcc" "ar" CMAKE_AR ${FILE_NAME_GCC})
+    string(REPLACE "gcc" "ranlib" CMAKE_RANLIB ${FILE_NAME_GCC})
+   else()
+    string(REPLACE "gcc" "gcc-ar" CMAKE_AR ${FILE_NAME_GCC})
+    string(REPLACE "gcc" "gcc-ranlib" CMAKE_RANLIB ${FILE_NAME_GCC})
```

Comment:
This is necessary for LTO build, but should be not necessary if LTO is turned off.

Do I understand correctly that LTO is not supported on OpenWRT?
If so, `ENABLE_LTO` should be turned off for OpenWRT builds and this `if` block should become unnecessary.
Could you, please, try building the target without adding the `if`, using `make LTO=OFF ...`?